nuget.config

```diff
@@ -0,0 +1,11 @@
+<configuration>
+    <config>
+        <add key="dependencyVersion" value="Highest" />
+        <add key="globalPackagesFolder" value="c:\packages" />
+        <add key="repositoryPath" value="packages" />
+    </config>
+    <packageSources>
+        <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+        <!--<add key="Local Source" value="D:\Data\Dev\LocalNuget" />-->
+    </packageSources>
+</configuration>
```

Comment:
This file is generated by a tool - i'm not sure it is a good idea to set a line wrap in, because quite probably it will keep pinging back when nuget touches it.

Reply:
Got it. Makes sense.